bin/host/src/fetcher/hint.rs

```diff
@@ -0,0 +1,72 @@
+//! This module contains the [HintType] enum.
```

Comment:
This will probably move into the client program, fyi. In here for now, but the `host` can depend on the `client`.